components/Behaviors/src/NavigateToUriBehavior.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xaml.Interactivity;
+
+namespace CommunityToolkit.WinUI.Behaviors;
+
+public sealed partial class NavigateToUriBehavior : DependencyObject, IAction
```

Comment:
Name should probably be `NavigateToUriAction` to align with other action names like [NavigateToPageAction](https://github.com/Microsoft/XamlBehaviors/wiki/NavigateToPageAction)